src/otx/algo/detection/necks/fpn.py

```diff
@@ -0,0 +1,227 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""FPN implementation."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from mmcv.cnn import ConvModule
+from torch import Tensor, nn
+
+if TYPE_CHECKING:
+    from mmengine import ConfigDict
+
+
+# This class come from mmdet and is slightly modified
+# https://github.com/open-mmlab/mmdetection/blob/ecac3a77becc63f23d9f6980b2a36f86acd00a8a/mmdet/models/necks/fpn.py
+class FPN(nn.Module):
```

Comment:
@jaegukhyun There's a duplication in instance segmentation. Could you remove the duplication in your next PR? Thanks.

Reply:
Sure.